keps/prod-readiness/sig-node/4205.yaml

```diff
@@ -4,3 +4,5 @@
 kep-number: 4205
 alpha:
   approver: "@johnbelamaric"
+beta:
+  approver: "@johnbelamaric"
```

Comment:
It would be worth checking with @johnbelamaric if he can take this one.

Otherwise I would post on #prod-readiness looking for an approver?

Reply:
maybe @deads2k? :)

Reply:
Yeah. I can get to this tomorrow